vcd/datasource_vcd_catalog_item_test.go

```diff
@@ -96,8 +96,8 @@ func catalogItemDestroyed(catalog, itemName string) resource.TestCheckFunc {
 
 func importStateIdByCatalogItem(objectName string) resource.ImportStateIdFunc {
```

Comment:
I think existing more generic function `importStateIdOrgObject` could be used.

Reply:
No it can't as it uses org:catalogname:objectname